### ISSUE TITLE
Add How To Guide for GTFS-RT

### ISSUE THREAD

**Issue by mrose914** [cal-itp/mobility-marketplace#451]
- [x] Confirm URL https://www.camobilitymarketplace.org/how-to/gtfs-rt
- [x] Add content to templated page 
- [x] Get approval on content

**Comment by mlgilbert18** [comment]
This is the first cut at the web page language. I added comments to explain the highlights. I'm working on creating this documents & getting them approved by DGS as we speak.[GTFS-Realtime "How To" web page.docx](https://github.com/user-attachments/files/18323810/GTFS-Realtime.How.To.web.page.docx)

**Comment by vykster** [comment]
@mlgilbert18 it looks like we might need to go back and forth on details in the content a bit. Please create a google doc and link here!

**Comment by vykster** [comment]
please also provide links to the pdf docs referenced

**Comment by mlgilbert18** [comment]
Sure, it is uploaded to the drive here: https://docs.google.com/document/d/1rlqpIqcVUboOLsIGH0xcWf8vx3jBq1df/edit?usp=sharing&ouid=108266805918927392972&rtpof=true&sd=true 

The PDFs are still with DGS for review and also cannot be shared until award given the information in them. But we are working to make that happen ASAP.

**Comment by ohmegasquared** [comment]
The WIP how to content is in designs in figma. Once we have the content, links, and PDFs locked down, I can share the designs for approval. 

@mlgilbert18 -  do you think we can finalize content by end of week?

**Comment by mlgilbert18** [comment]
website content yes, pdf content TBD. I have the "GTFS-Realtime into" deck as the main thing left to do. The other items are drafted in the folder but all require final changes based on the actual MSA # that DGS gives the contracts. We won't know that until award.

**Comment by mlgilbert18** [comment]
Oh and I do have 1 PDF (MSA overview) that is done but not uploaded because it contains info that still under NDA. BUT all that is approved by DGS and as soon as I get the green light from them it will be ready for upload.

**Comment by ohmegasquared** [comment]
[Figma](https://www.figma.com/design/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=6586-25707&t=qotVTIRQWcDZJlmT-1) updated with copy edits 

![Image](https://github.com/user-attachments/assets/932b41fb-ba72-4fd9-ab47-39139ba8a654)

**Comment by leelas-pm** [comment]
Hi @mlgilbert18 ! Hope you had a great vacation 😄 I just wanted to check in on the status of the content for this ticket - any updates on when we might have the green light from DGS on the pending content? Just wanted to do a bit of tracking from my end so we can factor it into planning. Thank you!

**Comment by mlgilbert18** [comment]
Hey, I made some comments on the Figma before I went out that I don't think have been addressed. DGS says award maybe this week or next but we don't have any more clarity than that. Other than the Figma edits, I believe the content should be ready but I will confirm tomorrow.

**Comment by leelas-pm** [comment]
Yes, so sorry - saw those right after I commented. Thank you! I think Meghan just tagged us both in the doc with a similar comment, but would it be ok if we standardized our workflow re: copyedits to go through Google Docs for all copyedits back and forth? Then once everything is final, moving everything over to Figma, and putting any additional comments in Github for visibility. Unfortunately commenting in Figma gets lost a bit, as we saw here. Does that approach work for you? Also, let me know if it would be helpful for you and I to set up some time! Last week was my first week on the websites work, so I appreciate that I may be missing a lot of nuances about what works best for you!

**Comment by mlgilbert18** [comment]
That's great -- happy to just do everything in google docs and then move it over. GitHub is not my strong suit but I am learning!

**Comment by leelas-pm** [comment]
Awesome! Yes, if Google Docs works for you, we can do the majority of our coordination there! I was a Jira and Smartsheet gal myself before joining Compiler, so we can skill up in GitHub together 😅

**Comment by leelas-pm** [comment]
@machikoyasuda here's the PDF! 

[GTFS RT MSA Info Packet.pptx.pdf](https://github.com/user-attachments/files/18971941/GTFS.RT.MSA.Info.Packet.pptx.pdf)

**Comment by leelas-pm** [comment]
URL updated to all lower case here too!
